### DISCUSSION TITLE
onDisconnect

### DISCUSSION THREAD

**Discussion by ackvf** [NoahZinsmeister/web3-react#552] [Q&A]
Hi, I need to change UI when user disconnects their wallet in metamask without interacting with my app.
  
Where can I listen to such events?

**Comment by svarcoder** [comment]
@ackvf Hiii, you can use account key for that. 
if account is undefinned then disconnect.

**Reply by ackvf** [reply]
@svarcoder Thank you for a response. You mean from the response object of `onDisconnect`, there is an `account` key? Could you point me to a knowledge base?
